src/assessments/automated-checks/build-test-steps-from-rules.tsx

```diff
@@ -53,7 +53,7 @@ function buildAutomatedCheckStep(rule: ScannerRuleInfo): TestStep {
         getInstanceStatus: getInstanceStatus,
```

Comment:
Does this file need to be renamed to something related to `requirement` rather than `test-step`? Just curios. Not a blocker.

Reply:
yes. I'm trying to rename one file at a time.